### ISSUE TITLE
Error build => spawn fortify-prepare ENOENT

### ISSUE THREAD

**Issue by jorgebaroncacv** [PeculiarVentures/fortify#254]
I have problems with the building of the app. 
When i launch "npm run build", i get the following error:
Error: spawn fortify-prepare ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)npm ERR! code ELIFECYCL

I supposed that i needed the fortofy-prepare library, but i do not kwon how to install it. I have already see in your repository that you have this library, but i had problems to including it to this proyect.

Am i doing something wrong in the process? 

Thaks so much.

**Comment by microshine** [comment]
Use this command
```
npm i https://github.com/PeculiarVentures/fortify-prepare -g
```

This is helper CLI application which allows copying required files only

**Comment by rmhrisk** [comment]
@jorgebaroncacv of course Fortify is OSS and you can fork it, however, we strongly recomend you use the shared implementation so you pick up the regular security fixes (for example in Electron), support for new languages, smart cards, and other features.

if there is something in the standard distribution you want and you dont see, please let us know with a bug tracking that request.

**Comment by jorgebaroncacv** [comment]
@microshine thanks for the answer. I tried it, but it did not work, because i needed the tsc libary to be installed. So i prefered to download fotify-web, include the tsc library and add it to the project, and i resolved this error. 

After i needed to intall electron-packager too.

I supposed that theses libraries are not installed, because they have to be installed globally before you build the project.

I also had to create the folder fortify_x64/opt/fortify.

Now i have problems with the file control inside the folder DEBIAN,because it does not exist, i am going to investigate it and try resolved it.

Thanks so much for your help.

@rmhrisk  I just started to investigate your library, i hope i could use it in my own project, so if i develop new features or fix bugs, i am going to report it because i expect that your project grow up.
